src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.targets

```
@@ -1,6 +1,11 @@
 <Project>
 
-  <ItemGroup Condition=" '$(UseMaui)' == 'true' or '$(UseMauiNuGets)' == 'true' or '$(UseMauiCore)' == 'true' ">
+  <!--
+    The IDE uses the @(ProjectCapability) items to activate various features.
+  -->
+
+  <!-- .NET MAUI telemetry and failure tracking -->
+  <ItemGroup Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' ">
     <ProjectCapability Include="UseMauiCore" />
```

Might read better with the following line to do:
```suggestion
  <ItemGroup>
    <ProjectCapability  Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' " Include="UseMauiCore" />
```

---

This was the original version - the merge is just bad:

https://github.com/dotnet/maui/commit/cca129b12482ba1b969c2d785a4629e23b126044#diff-3042f8c369245b07c4cbf2ce9ad84538a813ee7483e623863896611dc96f4edeR2-R5

This got me wondering if it was wrong and UseMauiCore should only be set in certain situations? I'll check with lutz.